covid19-etl/etl/idph_hospital_utilization.py

```diff
@@ -94,17 +94,25 @@ def parse_file(self, latest_submitted_date, url):
                 "province_state": self.state,
             }
             self.summary_locations.append(summary_location)
-
+            processed_dates = set()
+            duplicate_records = False  # Added temporarily to identify an intermittent issue of duplicate records
             for utilization in data:
+                reported_date = utilization["ReportDate"]
                 if (
                     latest_submitted_date is None
-                    or get_date_from_str(utilization["ReportDate"])
-                    > latest_submitted_date
-                ):
+                    or get_date_from_str(reported_date) > latest_submitted_date
+                ):  # TODO : Add logic to print the utilization records outside the loop if duplicates are found.
+                    if reported_date in processed_dates:
```

Comment:
your PR description says "Bug Fixes: Prevent submission of duplicate record" but i don't see where that's happening.
because we want to see more debug logs, we probably don't want to "bandaid fix" the issue yet

Reply:
* If a date is processed already, then we don't process the date again; It might be just a bandaid fix like you've mentioned. 
* I can go ahead and add a raise exception if there are duplicates while we print the `data` that we receive from the url to see what is going wrong. It won't be a fix in any way. But printing the whole response from the url would give us more info. 
* One thing I'm surprised is that, the job ran at 11:25 AM in the morning and failed, but the response of the url is having no issues when I checked later today. I was hoping these logs would help us confirm if there are any such issues from the source.

Reply:
> If a date is processed already, then we don't process the date again; It might be just a bandaid fix like you've mentioned.

I might be missing something, but I don't see where this is happening. But again, i don't think we should do it. I'm only saying we should update the PR description if we don't do it

Reply:
* Aahh! Shoot. I had a `continue` in the if condition in line 105. I must've accidentally deleted it when I added the boolean flag. 
* If you suggest against adding a bandaid fix, let us leave it the way it is, I will update the PR description. Next time we get an error we can hope we'd see more logs that might help us figure things out. 
* However does `Kibana` truncate the logs if they're too large?

Reply:
Kibana splits long logs into several rows